posthog/models/subscription.py

```diff
@@ -123,6 +121,33 @@ def url(self):
             return absolute_uri(f"/dashboard/{self.dashboard.id}/subscriptions/{self.id}")
         return None
 
+    @property
+    def summary(self):
+        human_frequency_map = {
```

Comment:
I wonder if it would read more nicely if you inlined these two variables...

They're both short